lib/ueberauth/strategy/facebook.ex

```diff
@@ -103,15 +103,17 @@ defmodule Ueberauth.Strategy.Facebook do
   """
   def info(conn) do
     user = conn.private.facebook_user
+    image_type = option(conn, :image_type)
 
     %Info{
       description: user["bio"],
       email: user["email"],
       first_name: user["first_name"],
-      image: fetch_image(user["id"]),
+      image: get_image(user, image_type),
       last_name: user["last_name"],
       name: user["name"],
       urls: %{
+        id: user["id"],
```

Comment:
Mainly because it's very handy. It is used in numerous api calls and is returned by default. (as far as my knowledge and tests go)